plugins/circleci/src/circleci-config.ts

```diff
@@ -70,11 +58,10 @@ export default abstract class CircleCiConfigHook extends Hook {
     // Automatically update tool kit orb if config is autogenerated
     const rawConfig = await this.getCircleConfigRaw()
     if (rawConfig && rawConfig.startsWith(automatedComment)) {
-      const targetTag = await this.getVersionTag()
       const currentOrb = config?.orbs['tool-kit']
       const currentTag = currentOrb && currentOrb.split('@')[1]
 
-      if (targetTag && targetTag !== developmentVersion && currentTag && targetTag !== currentTag) {
+      if (majorOrbVersion !== currentTag) {
```

Comment:
**question**: do you have a process in mind for managing these versions? should that be documented?

**question**: this seems to be checking the major version against the entire version number in the `config.yml`, is that right?

Reply:
I've now added the orb to the release-please manifest so it can be released just like our node packages (its version is tracked with a `version.txt` file.) Is that what you had in mind?

Fixed the bug when checking the major version too, thank you for spotting that!